ui/lib/core/addon/components/form-field.js

```diff
@@ -31,7 +31,8 @@ import layout from '../templates/components/form-field';
 export default Component.extend({
   layout,
   'data-test-field': true,
-  classNames: ['field'],
+  attributeBindings: ['data-test-field'],
```

Comment:
Why are we adding back attributeBindings here when we removed them everywhere else?

Reply:
I didn't glimmerize this component and now that automatic attribute binding has been removed defining `data-test-field: true` as a property does not do anything. Adding it to `attributeBindings` is necessary to bind it to the root element.

Reply:
Should we remove `data-test-field: true` then?

Reply:
I think there were instances where `data-test-field` was not passed into the component and some tests were failing so I added it back as a default. If the component value for `data-test-field` is not truthy the attribute will not be bound to the element.

Reply:
Thanks for the explanation! Can't wait till we glimmerize all the things.

Reply:
We must celebrate when that day arrives!